recipe/meta.yaml

```
@@ -18,23 +18,32 @@ source:
     # 2022/07 hmaarrfk
     # We've patch out the bug in GCC that makes it incompatible
     # https://github.com/conda-forge/ctng-compilers-feedstock/blob/7ea10c2526445beb9faff23efcc6bc40fc0079b5/recipe/meta.yaml#L23=
-    - patches/remove_gcc_maximum_requirements.patch 
+    - patches/remove_gcc_maximum_requirements.patch
 
 build:
-  number: {{ number }}
+  number: {{ number + 100 }}  # [blas_impl == "mkl"]
+  number: {{ number }}  # [blas_impl != "mkl"]
   skip: true  # [win]
```

If the user doesn't mind cuda, I guess they wouldn't mind MKL too?
```suggestion
  skip: true  # [cuda_compiler_version != "None" and blas_impl != "mkl"]
```

---

That seems fine but wouldn't this add windows builds back?

---

Fixed

---

Thank you!